test/core/test/happy-dom.test.ts

```diff
@@ -11,8 +12,19 @@ import { expect, it, vi } from 'vitest'
 declare global {
   // eslint-disable-next-line no-var
   var __property_dom: unknown
+  // eslint-disable-next-line no-var
+  var happyDOM: unknown
 }
 
+it('defaults URL to localhost:3000', () => {
```

Comment:
same applies here.

I'd like to try passing and not passing URL, but no idea how to do that using JSDoc comment.

Reply:
You can create a separate test file